File: Halloween_Candy/Halloween_Candy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int houses;
            houses = Convert.ToInt32(Console.ReadLine());
            
            //your code goes here
            double x = Math.Ceiling((2.0/houses)*100); 
     Console.WriteLine(x);
        }
    }
}

File: Halloween_Candy/Halloween_Candy.java
import java.util.Scanner;
import java.lang.Math;

public class Program
{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int houses = input.nextInt();
        
        //your code goes her
        
int x = (int) Math.ceil((2.0/houses)*100);
       
System.out.println(x);
    }
}
